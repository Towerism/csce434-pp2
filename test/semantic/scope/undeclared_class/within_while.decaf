int asdf() {
    int i;
    i = 0;
    while (i < 6) {
        Asdf fdsa;
        i++;
    }
}