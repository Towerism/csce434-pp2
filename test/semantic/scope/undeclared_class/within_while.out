
*** Error line 5.
        Asdf fdsa;
        ^^^^
*** No declaration found for type 'Asdf'

